ci/build.ps1

```diff
@@ -107,6 +107,9 @@ cmake -G $cmakeGenerator $(convertToCmakeArgs($nvimCmakeVars)) .. ; exitIfFailed
 cmake --build . --config $cmakeBuildType -- $cmakeGeneratorArgs ; exitIfFailed
 bin\nvim --version ; exitIfFailed
 
+# Ensure that win32 feature is set.
+bin\nvim -c 'exe has("win32") ? "q" : "cq"' ; exitIfFailed
```

Comment:
Nvim cq supports controlling the exit code.

    nvim -c "exe has('win32').'cq'" ; exitiffailed

Reply:
Thanks, amended.